### PR TITLE
add devicons, enhance iconlist

### LINKED ISSUE #41: Inclusion of devicons
Hi there,

Since 3.0 will give a Clients>Group>Projects folding, a wider set of icons would be awesome. Projects tend to be classified by their technology 

eg. Same Client. Website in Wordpress, admin tool in NodeJs, old Admin Tools in Python or Ruby.

[Devions ](https://github.com/vorillaz/devicons) are also used in a lot of other Atom plugins.

cheers ;)

### PR DESCRIPTION
**Pull request by Hammster**
closes #41

### CODE REVIEW

File: styles/project-viewer.less [hunk 1]
```diff
@@ -140,9 +147,18 @@ pv-remove-quick-modal {
...
 		cursor: pointer;
-		max-height: 100px;
+		max-height: 200px;
 		overflow-y: scroll;
 		overflow-x: hidden;
+
+		.btn{
+			width: 32%;
+			&:before{
+				float:left;
```
Comment (jccguimaraes) [inline styles/project-viewer.less]: OCD: please add a space between prop and value 👾

File: styles/project-viewer.less [hunk 2]
```diff
@@ -140,9 +147,18 @@ pv-remove-quick-modal {
...
+				width:20px;
```
Comment (jccguimaraes) [inline styles/project-viewer.less]: OCD: please add a space between prop and value 👾

File: src/modal-create-component.js [hunk 3]
```diff
@@ -62,19 +74,28 @@ function addIcons () {
...
     iconsList.classList.add('inline-block', 'pv-icons');
 
-    _octicons.icons.forEach(
-        (icon) => {
-            let entryIcon = document.createElement('button');
-            entryIcon.classList.add('btn', 'btn-sm', 'inline-block-tight', 'text-subtle', 'icon', icon);
-            entryIcon.textContent = icon;
-            entryIcon.addEventListener('click', addIconClickEvent.bind(this), false);
-            iconsList.appendChild(entryIcon);
+    let loopIcons = ( function (icon) {
```
Comment (jccguimaraes) [inline src/modal-create-component.js]: THOUGHT: can we make this as another function instead on an IIFE? and then call as `loopIcons.call(this, icon)`? what's your opinion?

**Comment by Hammster** [comment]
The pull request will be reopened since more things will be added

**Pull request closed without merging**